webknossos/examples/image_stack_to_dataset.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from webknossos import Dataset
+from webknossos.dataset import COLOR_CATEGORY
+
+INPUT_FILES = (
+    Path(__file__).parent.parent / "testdata" / "tiff" / "test.0000.tiff",
```

Comment:
Are these image files already part of the repo? Otherwise you would need to add them. If they are new files, you may want to rename them to indicate their source/origin, e.g. `test.cremi.0000.tiff` or something.